src/style.css

```diff
@@ -1,3 +1,93 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  color: black;
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  width: 1200px;
+  height: 768px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0 40px;
+}
+
+.header img {
+  width: 40px;
+  height: 40px;
+}
```

Comment:
Avoid using tag names for styling (except `html` and `body`). Style elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.